wp-content/themes/goonj-crm/functions.php

```diff
@@ -119,7 +119,12 @@ function goonj_check_user_action($atts)
     ob_start();
     $message = '';
     if (isset($_GET['message'])) {
-        if ($_GET['message'] === 'waiting-induction') {
+		if ($_GET['message'] === 'waiting-induction') {
+			echo '<style>
+					.user-identification-heading {
+						display: none;
+					}
+				</style>';
```

Comment:
Why are we echoing the CSS here and not writing in a CSS file?

Reply:
We need to apply this CSS based on the condition that the 'waiting-induction' parameter is present in the URL. If it is, the CSS should be appended to remove the heading.

Reply:
I don't understand the explanation. Why are we mixing CSS into PHP code is the simple question.

You said:
> We need to apply this CSS based on the condition that the 'waiting-induction' parameter is present in the URL

To do that we can do something like:

```php
// Read the param from the URL
$param = $_GET['param'];

$class = $param === 'waiting-induction' ? 'class-for-waiting-induction' : 'class-otherwise';

<div class="<?php echo $class; ?>
</div> 
```

In CSS we can have two different classes:
```css
.class-for-waiting-induction {
}

.class-otherwise {
}
```

Reply:
@pokhiii 
To give you content what I was doing here.
1. I have created a heading and subheading on the page itself where I wrote the code for something like this
<img width="808" alt="image" src="https://github.com/user-attachments/assets/30d0fae2-91a5-4f51-8c33-bdba9909886f">

2. A problem arises when the user is inducted but has not completed the induction so we are showing some message based on the params, `https://goonj.test/dropping-center-identification-form/?message=waiting-induction` so here the dropping centre message also came on screen as you see the below screenshot.
<img width="1592" alt="image" src="https://github.com/user-attachments/assets/bf3a0d14-cdff-438a-9253-8bf2580e3a3f">

3. **Mine approach:** To remove the message when the page is on the `?message=waiting-induction` I have added a CSS inside the condition where when the `$_GET['message'] === 'waiting-induction'` is true then display none the class with the help of this I was hiding the heading on another page.

As you have mentioned one approach above I believe it is helpful when we are showing the heading of the dropping center from the code itself.

Please let me know if is there anything I'm missing, I was trying to see if is there any approach to hide this without using CSS on my PHP.